packages/docs-app/src/examples/select-examples/multiSelectExample.tsx

```diff
@@ -176,4 +183,8 @@ export class MultiSelectExample extends BaseExample<IMultiSelectExampleState> {
             this.setState({ [prop]: event.currentTarget.checked });
         };
     }
+
+    private handleClear = () => {
+        this.setState({ films: [] });
```

Comment:
nit: small fns like this work great on one line. 
`private name = () => this.setState(...)`